src/main/java/org/gridsuite/study/server/StudyService.java

```diff
@@ -431,12 +434,18 @@ public StudyEntity doGetStudyWithPreFetchedLoadFlowResultAndUpdateIsPrivate(UUID
                     .body(BodyInserters.fromMultipartData(multipartBodyBuilder.build()))
                     .retrieve()
                     .onStatus(httpStatus -> httpStatus != HttpStatus.OK, clientResponse ->
-                            handleStudyCreationError(studyUuid, studyName, userId, isPrivate, clientResponse)
+                            handleStudyCreationError(studyUuid, studyName, userId, isPrivate, clientResponse, "case-server")
                     )
                     .bodyToMono(UUID.class)
                     .publishOn(Schedulers.boundedElastic())
                     .log(ROOT_CATEGORY_REACTOR, Level.FINE);
-        });
+        })
+            .onErrorResume(t -> {
+                if (!(t instanceof StudyException)) {
```

Comment:
Why this test ?

Reply:
To emit only once the error message, if handleStudyCreationError was called before (through onStatus)